fleetx/models/gpt_model/modeling.py

```diff
@@ -22,6 +22,7 @@
 from paddle.fluid import layers
 from paddle.nn.layer.transformer import _convert_param_attr_to_list
 import paddle.incubate as incubate
+from paddle.distributed.fleet.utils import recompute
 
 # import sys
```

Comment:
cleanup?

Reply:
``recompute`` is used in ``modeling.py``